### DISCUSSION TITLE
Search API issue (/cakestore/search)

### DISCUSSION THREAD

**Discussion by donghakang** [CAKK-official/CAKK_front#25] [General]
1. Notion, api, swagger 에서 search 다 다름… ➡️ addresses를 string으로 받아야하는가, 아니면 string[]으로 받아야하는가 (이게 옳다고봄)
2. cakestore/search => [‘null’] 이 아니면 internal server error ➡️ [] 가능하도록..
3. 모든게 null 이면 어떤 데이터를 가져오게 해야하는가...

**Comment by Hyevvy** [comment]
1. swagger를 수정해주겠다
2. [""]
3. 카테고리 address 둘 다 null 이면 모든 데이터를 주겠다
